analytics/tests/test_activity_views.py

```diff
@@ -218,3 +220,148 @@ def test_get_remote_server_guest_and_non_guest_count(self) -> None:
         remote_activity_counts = get_remote_customer_user_count(server_logs[server_id])
         self.assertEqual(remote_activity_counts.non_guest_user_count, 73)
         self.assertEqual(remote_activity_counts.guest_user_count, 16)
+
+    def test_remote_activity_with_robust_data(self) -> None:
+        def add_plan(customer: Customer, tier: int, fixed_price: bool = False) -> None:
+            if tier in (
+                CustomerPlan.TIER_SELF_HOSTED_BASE,
+                CustomerPlan.TIER_SELF_HOSTED_LEGACY,
+                CustomerPlan.TIER_SELF_HOSTED_COMMUNITY,
+            ):
+                price_per_license = 0
+            else:
+                price_per_license = 1000
+            if not fixed_price:
```

Comment:
Usually it's better to write `if fixed_price` as the first case -- just easier to read with fewer negations.

And then you can start the other case with `assert price_per_license > 0` or something in the block that's going to use that value.

Reply:
Yeah, we don't even need a price per license for the fixed price plan data. Revised so that the fixed_price true is the first case.